espflash/src/connection/mod.rs

```diff
@@ -163,6 +232,28 @@ impl Connection {
         Ok(())
     }
 
+    // Reset the device taking into account the reset after argument
+    pub fn reset_after(&mut self, is_stub: bool, chip: Chip) -> Result<(), Error> {
+        match self.after_operation {
+            ResetAfterOperation::HardReset => HardReset.reset(&mut self.serial),
+            ResetAfterOperation::SoftReset => {
+                println!("Soft resetting");
```

Comment:
These should probably be log statements, right?

Reply:
Updated to info messages